### PR TITLE
added in styles for overview page 

### PR DESCRIPTION
**Pull request by kathleenAMA**
- most styles went into base but some into typography (like heading styles)

@kaytcampbell

### CODE REVIEW

File: app/assets/stylesheets/ama_layout/modules/blue-boxes.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  background-repeat:no-repeat;
+  background-position:center center;
+}
+
+.box-title {
+  text-align: center;
+  color: white;
+  width: 90%;
+  margin: 10px auto 0 auto;
+}
```
Comment (kaytcampbell) [inline app/assets/stylesheets/ama_layout/modules/blue-boxes.scss]: Just missing some variables for the margins in this section.

File: app/assets/stylesheets/ama_layout/layout/ama-typography.scss [hunk 2]
```diff
@@ -38,9 +38,20 @@ small{
 }
 
 .uppercase-title {
-  font-size: 14px;
+  font-size: $base-font-size*0.875;
   font-weight: normal;
   color: #5C5C5C;
```
Comment (kaytcampbell) [inline app/assets/stylesheets/ama_layout/layout/ama-typography.scss]: Needs a colour variable.

**Comment by kaytcampbell** [comment]
:hankey: :hankey: :hankey:

**Comment by kathleenAMA** [comment]
:wave:

**Comment by kaytcampbell** [comment]
:art: :art: :art:

**Pull request merged**